docs/sequence.puml

```diff
@@ -0,0 +1,58 @@
+@startuml
+database Bitcoin
+actor Bob
+Actor Alice
+database Monero
+
+Bob --> Alice: Establish connection
+
+group Request Response Channel
+    Bob -> Alice: Request Amounts
+    note left: Btc Amount
+
+    Alice --> Bob: Send Amounts
+    note right: Btc & Xmr Amounts
+end
+
+group Request Response Channel
+    Bob -> Alice: bob::Message0
+    note left: Pubkeys\ndleq proof sb\nxmr viewkey vb\nbtc refund addr
+
+    Alice --> Bob: alice::Message0
+    note right: Pubkeys\ndleq proof sa\nxmr view key va\nbtc redeem addr\nbtc punish addr
+end
+
+group Request Response Channel
+    Bob -> Alice: bob::Message1
+    note left: btc lock tx
```

Comment:
What's the level of detail you want to have? Might be important to point out that this is not yet signed.

Reply:
Good to ensure it's correct, I am not going to modify the content of the execution part. The intent of this PR is to have a starting document from which I can plan the next steps:
- replace request amounts/amounts with appropriate messages for single market maker
- Replace request response channels with one shot messages